avaframe/com4FlowPy/com4FlowPyCfg.ini

```diff
@@ -72,6 +72,11 @@ forest = False
 
 # ['forestFriction', 'forestDetrainment', 'forestFrictionLayer']
 forestModule = forestFriction
+# ForestInteraction gives an additional output layer for the amount of forested cells
```

Comment:
so you mean I should rebase it with the other branch?
I guess the easiest way is to rebase it, when one of the PRs is merged to master?

Reply:
Yes, that's what I meant - maybe we can first merge PR #996, since this one only affects ```flowCore.py``` and then proceed with PR #995 or PR #997 - if this is ok with you ...

Reply:
- yes thats  good!

- Ah, yeah I understand! I will do these changes in the alpha/umax PR. honestly, I was just too lazy and took your forestFrictionLayer module for the variable alpha layer, as it is the same, more less ;)